pkg/tbtc/gen/pb/message.proto

```diff
@@ -7,4 +7,12 @@ message StopPill {
     uint64 attemptNumber = 1;
     string dkgSeed = 2;
     string messageToSign = 3;
+}
+
+message SigningSyncMessage {
```

Comment:
Also, what do you think about `signingSyncer`? Does your proposition involve just the message rename or the entire "sync" concept as well?

Reply:
Are we going to use this message for a single message case as well? I think it makes sense because we could cancel the context, stopping retransmissions and members excluded from the current attempt, once a confirmation message from all participants is received.

I think it is *sync* only in case of a batch signing, when we want to sync operators about the start block for the next message. 

I would personally prefer one of:
- `SigningStatusMessage` & `signingStatusCheck`,
- `SigningCompletedMessage` & `signingCompletedCheck`,
- `SigningDoneMessage` & `signingDoneCheck`.

But if you have a strong opinion that `SigningSyncMessage` & `signingSyncer` are better, I am okay with leaving it as-is. I just think that "signing status", "signing completed", "signing done" reflect a bit better what and when this thing is happening than "signing sync".

Reply:
> Are we going to use this message for a single message case as well

Yep, we are doing it right now. I like `SigningDoneMessage` & `signingDoneCheck` so I'll rename the current structure accordingly.

Reply:
See https://github.com/keep-network/keep-core/pull/3404/commits/95c219dc50a16c56aeb7b573e4a0ebb4b2f67167